.pre-commit-config.yaml

```diff
@@ -78,7 +75,7 @@ repos:
       - id: shfmt
         args: ["-i", "4"]
       - id: shellcheck
-        exclude: (deploy/deployment-manager/deploy.sh|deploy/deployment-manager/set_env.sh|.buildkite/scripts/publish.sh|dev-tools/packaging/docker/elastic-agent/build.sh|scripts/common.sh|scripts/make/common.bash)
+        exclude: (deploy/deployment-manager/deploy.sh|deploy/deployment-manager/set_env.sh|.buildkite/scripts/publish.sh|dev-tools/packaging/docker/elastic-agent/build.sh|scripts/common.sh|scripts/make/common.bash|scripts/bump_cloudbeat.sh|scripts/bump_integration.sh)
```

Comment:
why are you excluding your new scripts?

Reply:
that was a mistake. it was meant to be added to `files` but we've changed to used `exclude`. removed it.